.github/workflows/ci.yml

```
@@ -1,128 +1,269 @@
 name: Omega_KG CI Pipeline
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)